hbase-client/src/main/java/org/apache/hadoop/hbase/Abortable.java

```diff
@@ -37,6 +37,10 @@
    */
   void abort(String why, Throwable e);
 
+  default void abort(String why) {
```

Comment:
Please add a javadoc comment to mention that it will just call the other abort method with a null throwable parameter.

And also a `@see abort(xxxx)`

Reply:
done